### DISCUSSION TITLE
Kubernetes yaml manifest.

### DISCUSSION THREAD

**Discussion by x-real-ip** [immich-app/immich#1819] [General]
I had some trouble running the Immich application (containers) under Kubernetes. The Helm chart got me started but I don't like Helm and I've tweaked it here and there. For those who want to run it under Kubernetes [here](https://github.com/theautomation/gitops/blob/main/manifests%2Fimmich%2Fbase%2Fmanifest.yaml) you can find my yaml manifest, take advantage of it. The more people running this application the more support and hopefully donations and contributions for the developers. Thank you for this great application. Perhaps the yaml manifest could be included in the documentation so that one has a choice between Helm and a raw yaml manifest.

**Comment by jasjeetsuri** [comment]
Can you share that? I’m getting a 404

**Reply by x-real-ip** [reply]
https://github.com/theautomation/gitops/blob/main/manifests%2Fimmich%2Fbase%2Fmanifest.yaml
